admin/app.js

```diff
@@ -133,9 +133,9 @@ if (config.Logging.Express.UseWinston === 'true') {
 
 app.use(cors())
 app.use(helmet())
-const scriptSources = ["'self'", "'unsafe-inline'", 'https://www.google-analytics.com']
+const scriptSources = ["'self'", "'unsafe-inline'", 'https://www.google-analytics.com', 'https://www.googletagmanager.com']
 const styleSources = ["'self'", "'unsafe-inline'"]
-const imgSources = ["'self'", 'https://www.google-analytics.com', 'data:']
+const imgSources = ["'self'", 'https://www.google-analytics.com', 'https://www.googletagmanager.com', 'data:']
```

Comment:
@GuyHarwood doubt we need it here.
Why was even analytics in the imgSources in the first place? 🤔

Reply:
because it drops pixels https://developers.google.com/analytics/resources/concepts/gaConceptsTrackingOverview